bootstrap/aws/config-default.sh

```diff
@@ -13,7 +13,8 @@ ZONE=${APOLLO_AWS_ZONE:-eu-west-1b}
 MASTER_SIZE=${MASTER_SIZE:-m1.medium}
 SLAVE_SIZE=${SLAVE_SIZE:-m1.medium}
 NUM_SLAVES=${NUM_SLAVES:-1}
-
+FRAMEWORK_MARATHON_ENABLED=${FRAMEWORK_MARATHON_ENABLED:-'Y'}
```

Comment:
Completely agree. I'll move to its own file the plugins settings. We can do more refactor later.
Ideally/eventually the only settings inside /some-cloud/config-default.sh should be the credential ones